neo4j-plugin/docs/Neo4jSink-batchsink.md

```diff
@@ -0,0 +1,49 @@
+# Neo4j Batch Sink
+
+
+Description
+-----------
+Write data to Neo4j instance using a configurable CQL query.
+Count of created Node and Relation depend on Output Query.
+
+
+Use Case
+--------
+The sink is used whenever you need to write to a Neo4j instance.
+
+
+Properties
+----------
+**Reference Name:** This will be used to uniquely identify this source for lineage, annotating metadata, etc.
+
+**Neo4j Host:** Neo4j database host.
+
+**Neo4j Port:** Neo4j database port.
+
+**Output Query:** The query to use to export data to the Neo4j database. Query example: 'CREATE (n:<label_field> $(*))' 
```

Comment:
please clarify how to use properties and wildcards or add a reference to some docs.

Reply:
Done.